addons/im_livechat/static/src/embed/common/message_patch.js

```diff
@@ -19,6 +19,6 @@ patch(Message.prototype, {
      * @param {import("@im_livechat/embed/common/chatbot/chatbot_step_model").StepAnswer} answer
      */
     answerChatbot(answer) {
-        return this.props.message.thread.post(answer.name);
+        return this.props.message.thread.post(answer.name, {}, { selected_answer_id: answer.id });
```

Comment:
This is a FIX so that's fine.
Personally I would expect answering chatbot to not be an actual message post. The UI simulates it but it would feel more natural as a RPC to select the appropriate answer.

_(The concern is that this complexifies the `message_post` param list for not much reasons... We don't support answering a question of chatbot by typing in composer so that looks like a semblance of engineering)_